package.json

```diff
@@ -4,8 +4,10 @@
   "description": "This is the visit africa backend",
   "main": "src/server.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "start-dev": "nodemon --exec babel-node src/server.js"
+    "test": "./node_modules/.bin/mocha --require @babel/register src/test/*.js --exit",
```

Comment:
I keep saying that you shouldn't change this line as it will be changed on it own PR. Leave it as it is.

Reply:
Yeah No problem I am letting it as it is